packages/puppeteer-core/src/bidi/core/BrowsingContext.ts

```diff
@@ -503,8 +503,8 @@ export class BrowsingContext extends EventEmitter<{
     } = await this.#session.send('storage.getCookies', {
       ...options,
       partition: {
-        type: 'context',
-        context: this.id,
+        type: 'storageKey',
```

Comment:
would not it fetch all cookies in the user context instead of only cookies related to the current browsing context?

Reply:
Would it include partitioned cookies set for Chrome?

Reply:
> would not it fetch all cookies in the user context instead of only cookies related to the current browsing context?

It would, and it is consistent with the current behavior of BiDi cookies.

Reply:
It does not look like the expected behavior from the Puppeteer perspective? Do we have a test for partitioned cookies?

I think if this is the intention, I suggest we instead use the method on UserContext in Puppeteer instead of changing calls here.

Reply:
I'm not sure what "related to the current browsing context" means. Isn't it something like https://github.com/w3c/webdriver-bidi/issues/645, which we decided not to implement?